test/scripts/e2e-cp.sh

```diff
@@ -183,3 +185,32 @@ go test -test.timeout 180m -v . \
         ${NUM_NODES:+"--num-nodes=${NUM_NODES}"} \
         ${E2E_REPORT_DIR:+"--report-dir=${E2E_REPORT_DIR}"}
 popd
+
+if [[ "${WHAT}" == "${NETWORK_SEGMENTATION_TESTS}"* ]]; then
+  pushd "$TMP_DIR"
+  curl -L https://github.com/pperiyasamy/kubernetes/raw/netpol-udn-release-1.30/test/e2e/kubernetes-test-linux-amd64.tar.gz -o kubernetes-test-linux-amd64.tar.gz
+  tar xvzf kubernetes-test-linux-amd64.tar.gz
+  sudo mv e2e.test /usr/local/bin/e2e.test
+  rm kubernetes-test-linux-amd64.tar.gz
+  popd
+
+  export NUM_WORKER_NODES=3
+  if [ "$SINGLE_NODE_CLUSTER" == true ]; then
+    export NUM_WORKER_NODES=1
+  fi
+
+  # NetPol with named port is not supported in ovn-kubernetes, so skip those tests.
+  ginkgo --nodes=${NUM_NODES} \
+	  --focus="Netpol" \
+	  --skip="named port.+\[Feature:NetworkPolicy\]" \
```

Comment:
Why do we need this when we already have 
https://github.com/ovn-org/ovn-kubernetes/blob/master/test/scripts/e2e-kind.sh#L28-L29

Reply:
the e2e-kind.sh script does not run for network segmentation lane instead e2e-cp.sh script is [invoked](https://github.com/ovn-org/ovn-kubernetes/blob/master/.github/workflows/test.yml#L551) for it. hence another skip for named port netpol tests here.

Reply:
But is that is not running conformance tests? I am not aware that we have any `named port.+\[Feature:NetworkPolicy\]` in our control plane tests but I might be missing something.

Reply:
We discussed not adding this in this PR for the time being.